pkg/synthetictests/duplicated_events.go

```diff
@@ -185,9 +185,16 @@ var knownEventsBugs = []knownProblem{
 		BZ:       "https://bugzilla.redhat.com/show_bug.cgi?id=2017435",
 		Topology: topologyPointer(v1.SingleReplicaTopologyMode),
 	},
+	// builds tests trigger many changes in the config which creates new rollouts -> event for each pod
+	// working as intended (not a bug) and needs to be tolerated
+	{
+		Regexp:    regexp.MustCompile(`ns/openshift-route-controller-manager deployment/route-controller-manager - reason/ScalingReplicaSet \(combined from similar events\): Scaled (down|up) replica set route-controller-manager-[a-z0-9-]+ to [0-9]+`),
+		TestSuite: stringPointer("openshift/build"),
+	},
+	// builds tests trigger many changes in the config which creates new rollouts -> event for each pod
+	// working as intended (not a bug) and needs to be tolerated
```

Comment:
If using a daemonset that causes failures during draining as leases are not gracefully released and instead age out as pods are killed after potentially losing network access due to daemonset pods not being terminated is not a bug, what is it?  We know the better solution and fixed other components like the openshift-apiserver to use it.

Reply:
What about this event implies "failures during draining" or leases not being gracefully released?

The event in question is that a pod gets deleted, which is a result of the cluster config changing(as explicitly changed by a test) and so the controller manager does a rollout in response to the config change.

The test seems to assume we should never see those pods deleted (which, at steady state w/ no config changes, is probably true)

Reply:
@atiratree this might be worth reporting as a bug which we might address later on.